scripts/import_iris.py

```diff
@@ -0,0 +1,34 @@
+import geopandas as gpd
+from geoalchemy2.shape import from_shape
+from geopandas import GeoDataFrame, GeoSeries
+from shapely.geometry import Point, Polygon
+
+from models import IrisFrance
+from repository import repository
+
+
+def import_iris_shape_file_to_table(file_path: str):
+    filtered_iris_data = read_iris_shape_file(file_path)
+    for iris_row in filtered_iris_data.iterrows():
+        fill_iris_from(iris_row[1])
+
+
+def read_iris_shape_file(file_path: str) -> GeoDataFrame:
+    iris_data = gpd.read_file(file_path)
+    filtered_iris_data = iris_data[['CODE_IRIS', 'geometry']]
+    filtered_iris_data = filtered_iris_data.to_crs({'init': 'epsg:4326'})
```

Comment:
Je ne suis pas sûr de comprendre cette ligne facilement :/

Reply:
On attribue le référentiel géographique WGS aux IRIS (= référentiel GPS, centré sur (lat 0, lon 0))
```suggestion
    filtered_iris_data_in_wgs_format = filtered_iris_data.to_crs({'init': 'epsg:4326'})
```